Userland/Applications/Spreadsheet/Readers/XSV.h

```diff
@@ -107,7 +107,7 @@ class XSV {
         size_t index() const { return m_index; }
         size_t size() const { return m_xsv.headers().size(); }
 
-        using ConstIterator = AK::SimpleIterator<const Row, const StringView>;
+        using ConstIterator = AK::SimpleIterator<const Row, StringView const>;
```

Comment:
curiously, it didn't format this as `AK::SimpleIterator<Row const, StringView  const>`. nor below in the constructor of RowIterator, `RowIterator(const XSV&`

Reply:
Apparently there's still 100+ of these, sigh. Will not fix those up by hand, that's for the clang-format people to fix.

![image](https://user-images.githubusercontent.com/19366641/204536909-f45010fd-f1ad-4ad5-b89b-d140e6d7bf3c.png)